src/inspector_socket_server.cc

```diff
@@ -82,10 +82,10 @@ void PrintDebuggerReadyMessage(const std::string& host,
     return;
   }
   fprintf(out,
-          "Debugger listening on port %d.\n"
+          "Debugger listening on %s:%d.\n"
```

Comment:
I do need the change here for the tests to work correct. I suspect the last one to land will need to resolve some merge conflicts.